CoffeaAnalysis/corrections/muon/Highpt_muon_trigger_eff.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
For the moment us the available SFs which are closest to our selection. Because we planning to do OR between trigger paths, we'll need to rederive SFs. So for the moment it is ok if SFs are measured for a slightly different configuration than ours.